README.md

```diff
@@ -74,7 +74,7 @@ required for these tests.
 
 ```
 $ pip install pipenv
-$ pipenv --two install -r requirements.txt
+$ pipenv --two install
```

Comment:
This should be even easier now. Try simply `pipenv run pytest` and the virtual environment should be created if if doesn't already exist.